### ISSUE TITLE
Can't remove uninstalled PWAs from Control Panel/Programs

### ISSUE THREAD

**Issue by Rexadev** [filips123/PWAsForFirefox#244]
### Steps to Reproduce
1. In WIndows GO to Control Panel\Programs\Programs and Features
2. Uninstalled PWAs are still there

### Environment

* Operating system: Windows
* System architecture:
* PWAsForFirefox extension version: Latest
* PWAsForFirefox native version: Latest
* Firefox version: Latest

**Comment by filips123** [comment]
How did you uninstall those PWAs? If using profile deletion, this is a known bug that will be fixed in the next release. Also, can you check if they are still listed in `firefoxpwa profile list`?

To remove them, you will have to manually delete some registry keys:
1. Open Registry Editor (regedit).
2. Open `HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall` and delete `FFPWA-` subkeys that match your deleted PWAs.
3. Do the same for `HKCU\SOFTWARE\FirefoxPWA`'s and `HKCU\SOFTWARE\Classes`' subkeys.
4. Open `HKCU\SOFTWARE\RegisteredApplications` and delete `FFPWA-` values that match your deleted PWAs.

**Comment by Rexadev** [comment]
> Also, can you check if they are still listed in firefoxpwa profile list?

No.

> HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall

It also has old firefox versions



Also I am not able to uninstall old Profiles

**Comment by filips123** [comment]
> It also has old firefox versions

Maybe you misunderstood me. I meant literally string "CurrentVersion". Open regedit and copy `HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall` into its address bar. Then, you will see a bunch of `FFPWA-` subkeys in the tree on the left. You can click on each of them to check which PWA they are for, and delete unwanted ones.

> Also I am not able to uninstall old Profiles

What happens when you try to uninstall them?

**Comment by Rexadev** [comment]
> Maybe you misunderstood me.

I understand you. I am saying there is also old version of firefox like iirc ther was firefox 105 and 106

> What happens when you try to uninstall them?

It shows `removed!` but when I click on addon again it is still there

**Comment by filips123** [comment]
Can you check logs? Steps for obtaining logs are [here](https://github.com/filips123/PWAsForFirefox/wiki/Troubleshooting).

**Comment by Rexadev** [comment]
Log is showing wrong time but I copied some lnes from the bottom of the log
```
Caused by:
    0: Failed to flush stdout
    1: The pipe is being closed. (os error 232)
12:23:59 [WARN] Default profile cannot be completely removed
12:23:59 [WARN] Web apps and data will be cleared, but the profile will stay
12:24:13 [WARN] Default profile cannot be completely removed
12:24:13 [WARN] Web apps and data will be cleared, but the profile will stay
06:40:16 [ERROR] Failed to receive request

Caused by:
    0: Failed to read message size
    1: failed to fill whole buffer
06:40:16 [ERROR] Failed to send response

Caused by:
    0: Failed to flush stdout
    1: The pipe is being closed. (os error 232)
04:41:33 [ERROR] Failed to receive request

Caused by:
    0: Failed to read message size
    1: failed to fill whole buffer
04:41:33 [ERROR] Failed to send response

Caused by:
    0: Failed to flush stdout
    1: The pipe is being closed. (os error 232)
14:38:18 [WARN] Default profile cannot be completely removed
14:38:18 [WARN] Web apps and data will be cleared, but the profile will stay
14:38:30 [WARN] Default profile cannot be completely removed
14:38:30 [WARN] Web apps and data will be cleared, but the profile will stay
14:38:48 [WARN] Default profile cannot be completely removed
14:38:48 [WARN] Web apps and data will be cleared, but the profile will stay
14:56:21 [WARN] Default profile cannot be completely removed
14:56:21 [WARN] Web apps and data will be cleared, but the profile will stay
14:57:13 [WARN] Default profile cannot be completely removed
14:57:13 [WARN] Web apps and data will be cleared, but the profile will stay
14:58:00 [WARN] Default profile cannot be completely removed
14:58:00 [WARN] Web apps and data will be cleared, but the profile will stay
14:58:08 [WARN] Default profile cannot be completely removed
14:58:08 [WARN] Web apps and data will be cleared, but the profile will stay
13:39:22 [WARN] Default profile cannot be completely removed
13:39:22 [WARN] Web apps and data will be cleared, but the profile will stay
17:40:46 [WARN] Default profile cannot be completely removed
17:40:46 [WARN] Web apps and data will be cleared, but the profile will stay
18:50:02 [WARN] Default profile cannot be completely removed
18:50:02 [WARN] Web apps and data will be cleared, but the profile will stay
04:13:26 [WARN] Default profile cannot be completely removed
04:13:26 [WARN] Web apps and data will be cleared, but the profile will stay
04:13:39 [WARN] Default profile cannot be completely removed
04:13:39 [WARN] Web apps and data will be cleared, but the profile will stay
04:14:41 [WARN] Default profile cannot be completely removed
04:14:41 [WARN] Web apps and data will be cleared, but the profile will stay
```

**Comment by filips123** [comment]
Fix for #243 that has been released in 2.2.0 may also fix this. Please update to that version and check if it works.
